app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -104,24 +103,24 @@ UK ENIC can give you a [statement of comparability](https://www.enic.org.uk/Qual
 
 * help you can get with your application 
 
-If you intend to apply for teacher training, you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain experience until your teacher training course starts. 
+If you intend to apply for teacher training, you could consider [finding work as a teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain experience until your teacher training course starts. 
 
 ### Fees and financial support
 
 Ukrainian refugees in the UK under a Ukraine visa scheme have access to the same financial support as UK applicants. This means you'll:
 
-* have the right to apply for a [student loan, bursary, scholarship](/funding-and-support) or an [advanced learner loan](https://www.gov.uk/advanced-learner-loan) to help cover your fees and living expenses
+* have the right to [apply for a student loan, bursary, scholarship](/funding-and-support) or an [advanced learner loan](https://www.gov.uk/advanced-learner-loan) to help cover your fees and living expenses
```

Comment:
Should the second link be 'apply for an advanced...' to make it consistent for screen readers?